tests/foreman/cli/test_activationkey.py

```diff
@@ -183,7 +183,7 @@ def test_positive_create_with_usage_limit_default_explicitly(self):
 
         @Assert: Activation key is created
         """
-        new_ak = self._make_activation_key({u'unlimited-content-hosts': '1'})
+        new_ak = self._make_activation_key()
```

Comment:
Hm... But the test name says `create_with_usage_limit_default_explicitly`. If you remove parameter here it won't be explicit anymore.
`unlimited-content-hosts` is a flag without any value now, right? Why not try to `self._make_activation_key({u'unlimited-content-hosts': True})`?